src/store/reducers/workplaceReducer.ts

```diff
@@ -51,6 +67,7 @@ export interface Order {
     master: Master,
     user: User,
     master_busyDate: MasterBusyDate,
+    orderPictures: OrderPicture[] | []
```

Comment:
OrderPicture[] | [] ?

Reply:
if there is no image in the order, the backend returns an empty array